Snakefile-sorting-hat

```diff
@@ -190,12 +183,11 @@ rule produce_recalculations_call:
         pushd {input.working_directory}
 
 
-        snakemake --forceall --latency-wait 150 {config['sm_options']} --config \
+        snakemake --latency-wait 150 {config['sm_options']} --config \
```

Comment:
How are we passing the forceall behaviour to the Snakefile analysis part? When the sorting hat receives the command to force, then the subworkflow needs to apply it as well, otherwise older results (from a previous release, won't be re-written).

Reply:
In the current implementation, `--forceall` is passed here `{config['sm_options']}`  to the snakemake call in the sorting hat.